nixos/doc/manual/development/activation-script.section.md

```diff
@@ -57,7 +57,8 @@ There are some snippets NixOS enables by default because disabling them would
 most likely break your system. This section lists a few of them and what they
 do:
 
-- `binsh` creates `/bin/sh` which points to the runtime shell
+- `usrbin` creates `/usr/bin/env` and `/bin/sh` (which points to the runtime
```

Comment:
/usr/bin/sh then?

Reply:
The path we *want* to create is `/bin/sh`, the fact that the real path is `/usr/bin/sh` is an implementation detail.

Reply:
I'm not too sure, it just doesn't read cleanly to me.

Reply:
An alternative might be to rename the activation script `impurePaths` or something. Saying "we create `/usr/bin/sh` which, by the way, ends up in `/bin/sh`" feels very backwards to me.